core/services/repo_manager.rb

```diff
@@ -51,6 +52,13 @@ def find_repository(product_name, product, box)
       repo_key = "#{repository_name}@#{version}+#{repository_key}"
       repo = @repos[repo_key]
     end
+    if product.key?('version') && repo.nil?
+      repo = find_last_repository_by_major_version(product, repository_key)
+      unless repo.nil?
+        @ui.warning('MDBCI uses the minor version for the specified major version of the product!')
```

Comment:
MDBCI could not find the specified version VERSION HERE, automatically using the closes version VERSION HERE>